lib/livebook/runtime/erl_dist/manager.ex

```diff
@@ -212,6 +230,25 @@ defmodule Livebook.Runtime.ErlDist.Manager do
     {:noreply, state}
   end
 
+  def handle_cast(
+        {:request_completion_items, send_to, ref, hint, container_ref, evaluation_ref},
+        state
+      ) do
+    if evaluator = Map.get(state.evaluators, container_ref) do
+      Evaluator.request_completion_items(evaluator, send_to, ref, hint, evaluation_ref)
+    else
+      # Since there's no evaluator, we may as well get the completion items here.
+      spawn_link(fn ->
```

Comment:
If this fails for whatever reason, it will make the current process crash. Is this what we want? Are the evaluator processes also linked to the manager process? We probably want to spawn it all under a supervisor instead.

Reply:
Good point! Evaluators are supervised, but so should this completion task. I added a dedicated `Task.Supervisor` for that. I also noticed that manager doesn't shutdown upon receiving `:EXIT`, so I fixed that (we trap exists, because if one of the linked supervisors fails, we still want to cleanup the node). Let me know if this makes sense =)